zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,330 @@
+    def close(self):
+        self.image.close()
+        logger.debug("%s closed" % str(self.image))
+
+@staticmethod
+def get_upload_param(req_header):
+    # type: (dict[str, str]) -> UploadParam
+
+    def get_long_field(key, default=None):
+        v = req_header.get(key, default)
+        try:
+            lv = long(v)
```

Ruff (0.13.1)

Undefined name `long`

(F821)